pkg/secrets/conjur/conjur_secrets_fetcher.go

```diff
@@ -0,0 +1,37 @@
+package conjur
+
+import (
+	"fmt"
+)
+
+type ConjurProvider interface {
+	RetrieveSecret(string) ([]byte, error)
+	RetrieveBatchSecrets([]string) (map[string][]byte, error)
+}
+
+type ConjurSecretsFetcherInterface interface {
```

Comment:
Please provide a description for this interface ideally with real-world example usage